src/harness/compilerImpl.ts

```diff
@@ -257,7 +257,6 @@ export function compileFiles(host: fakes.CompilerHost, rootFiles: string[] | und
     }
 
     // establish defaults (aligns with old harness)
-    if (compilerOptions.target === undefined && compilerOptions.module !== ts.ModuleKind.Node16 && compilerOptions.module !== ts.ModuleKind.NodeNext) compilerOptions.target = ts.ScriptTarget.ES3;
     if (compilerOptions.newLine === undefined) compilerOptions.newLine = ts.NewLineKind.CarriageReturnLineFeed;
     if (compilerOptions.skipDefaultLibCheck === undefined) compilerOptions.skipDefaultLibCheck = true;
     if (compilerOptions.noErrorTruncation === undefined) compilerOptions.noErrorTruncation = true;

```

Comment:
I want to say that we should drop these too, but:

- If we change the default `newLine`, all baselines will change from CRLF to LF and lose git history.
- `skipDefaultLibCheck` is a perf optimization.
- `noErrorTruncation` gives us better baselines.